lua/lspconfig/util.lua

```diff
@@ -272,7 +272,7 @@ function M.server_per_root_dir_manager(_make_config)
       end)
 
       -- Launch the server in the root directory used internally by lspconfig, if applicable
-      new_config.cwd = root_dir
+      new_config.cmd_cwd = root_dir
```

Comment:
We should also probably check if cmd_cwd is set in the prior config before overriding it

Reply:
Right, good idea, done (and removed it from julia).